sdk/purview/purview-datamap-rest/test/public/entityTest.spec.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+import { createClient } from "./utils/recordedClient";
+import { createFileFromStream } from "../../src/index";
+import fs from "fs";
+
+
+describe("purview datamap entity test", () => {
+  let recorder: Recorder;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("Import entity business metadata ", async function () {
+	const client = await createClient(recorder);
+   
+	const response = await client.path("/atlas/v2/entity/businessmetadata/import").post({
+	contentType: "multipart/form-data",
+	body: {
+		file: createFileFromStream(() => fs.createReadStream("test/public/template_2.csv"), "template_2.csv"),
+	}
```

Comment:
@timovv Any suggest if we want to allow the same code to run on both browser and node env?

Reply:
I would suggest inlining the file content and using `createFile` instead, something along the lines of:

```ts

const fileContent = new TextEncoder().encode(`TypeName,UniqueAttributeValue,BusinessAttributeName,BusinessAttributeValue,UniqueAttributeName[optional]
hive_database,hive_db_1,bmWithAllTypes.attr8,"Awesome Attribute 1",name`);

```

and then using `file: createFile(fileContent, "template_2.csv")`

Reply:
Updated. Thanks